### PR TITLE
SPEC-666: Add readable/writable rules table

### PR DESCRIPTION
**Pull request by durran**

### CODE REVIEW

**Comment by ajdavis** [comment]
Very nice, LGTM.

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst [hunk 1]
```diff
@@ -330,16 +330,49 @@ API
...
     /**
-     * Determines if the topology has a writable server available.
+     * Determines if the topology has a writable server available. See the table in the
+     * following section for behaviour rules.
      */
     hasWritableServer(): Boolean
   }
 
+---------------------------------------------------
+Determining If A Topology Readable/Writable Servers
```
Comment (behackett) [inline source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst]: has Readable/Writable

File: source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst [hunk 2]
```diff
@@ -330,16 +330,49 @@ API
...
+---------------------------------------------------
+
+The following table describes the behaviour of determining if a topology type has readable or
+writable servers. If no read preference is passed to ``hasReadableServer``, the driver MUST default
+the value to the default read preference, ``primary`` or treat the call as if ``primary`` was provided.
```
Comment (behackett) [inline source/server-discovery-and-monitoring/server-discovery-and-monitoring-monitoring.rst]: ``primary``,

**Comment by behackett** [comment]
Other than those two nits, LGTM.

**Comment by durran** [comment]
@behackett Fixed the grammar issues.

**Comment by behackett** [comment]
LGTM

**Pull request merged**